src/App.js

```diff
@@ -4,15 +4,57 @@ import './App.css';
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <h1>
-          Eric F. Olsen
-        </h1>
-        <h3>
-          Software Engineer
-        </h3>
-      </header>
+      <Header />
+      {/* <Story />
+      <Resume />
+      <Projects />
+      <Testimonials /> */}
+    </div>
+  );
+}
+
+function Header() {
+  return (
+    <header className="App-header">
+      <img src={logo} className="App-logo" alt="logo" />
+      <h1>
+        Eric F. Olsen
+      </h1>
+      <h3>
+        Software Engineer
+      </h3>
+    </header>
+  );
+}
+
+function Story() {
```

Comment:
This can be another file as well.  A function that can be used this way in React is typically referred to as a "component".

Reply:
Components are often all put under a "components" directory (but not required).

Reply:
Thanks for the suggestion! I've updated this branch to now have components in a "components" directory and in their own files.